harness/determined/keras/_tf_keras_trial.py

```diff
@@ -459,13 +459,6 @@ def _configure_callbacks(self, user_callbacks: Optional[List]) -> None:
             # Our implementation of verbose=True.
             callbacks = [keras.callbacks._DeterminedProgress()] + callbacks
 
-        profiler = keras.callbacks._DeterminedProfiler(
```

Comment:
Reminder here to add `Breaking Changes` to your release note.

Reply:
this in particular shouldn't be a breaking change, it's an internal callback. but my plan is to add release notes along with the feature branch. thanks for the reminder.